src/workflows-app/components/DeleteWorkflowModal.ts

```diff
@@ -0,0 +1,42 @@
+import { Modal, useThemeFromContext } from '@terra-ui-packages/components';
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ButtonPrimary } from 'src/components/common';
+import { icon } from 'src/components/icons';
+
+type DeleteWorkflowModalProps = {
+  methodName: string;
+  onDismiss: () => void;
+  onDelete: () => void;
+};
+
+export const DeleteWorkflowModal = ({ methodName, onDismiss, onDelete }: DeleteWorkflowModalProps) => {
```

Comment:
Did you take a look at `DeleteConfirmationModal` to see if you could use that?

Reply:
It would be great if we can reuse the component.

Reply:
Thanks for pointing that out!

Reply:
this was easy - I should have figured something like this existed already